modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -272,6 +272,9 @@ Other parser options let you:
 * Generate 999 Implementation Acknowledgments, rather than the 997 Functional Acknowledgments generated by default. Note that the support for 999 Implementation Acknowledgments does not include CTX segment generation.
 * Report segment error details to sender in the generated 997/999 acknowledgments.
 * Automatically include a standard 997 or 999 schema that matches the generated acknowledgment in the configuration for received messages.
+* Set the path to a Functional Acknowledgment schema to be used for generated acknowledgments such as `/hipaa/005010X231A1/999.esl` which uses the standard HIPAA 005010X231A1 acknowledgment.
+ Using this means the schema type (997 or 999) must match the Generate 999 Functional Acknowledgments setting (false for 997, true for 999).
+ If not set, the parser uses a hardcoded X12 997 or 999 schema, as appropriate.
 
```

Comment:
I looked at the output for this file and noticed several issues with title capitalization and list-item format.
So...this whole thing could use a copyedit at some point.
Does this deliverable follow the connector template?